boards/arm/mec1501modular_assy6885/doc/index.rst

```diff
@@ -158,6 +181,19 @@ This is the recommended setup.
 |             |     1      |  Use 3.3V Shared channel(R)|
 +-------------+------------+----------------------------+
 
+Power management.
```

Comment:
Would you remove the period at the end of this heading (and the "Boot-ROM Straps." above and "Jumper location map." below (Headings shouldn't end with a period).

Reply:
Periods removed for all these headings